### BIG differences, this is diff of wrong files.
### CLUS "Restart STORE-VALUE"
### and CLHS topic selection http://clhs.lisp.se/Body/a_store_.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:restarts:store-value.txt	2017-04-27 22:00:01.000000000 +0100
+++ saved-pages-clus/cl:restarts:store-value.txt	2017-04-27 22:00:01.000000000 +0100
@@ -1,4 +1,47 @@
-STORE-VALUE
-Please select which reference to STORE-VALUE you intended
+Restart STORE-VALUE

-FunctionRestart
\ No newline at end of file
+Data Arguments Required
+a value to use instead (on an ongoing basis).
+Description
+The store-value restart is generally used by handlers trying to recover from errors of types such as cell-error or type-error, which may wish to supply a replacement datum to be stored permanently.
+Examples
+(defun type-error-auto-coerce (c)
+  (when (typep c 'type-error)
+    (let ((r (find-restart 'store-value c)))
+      (handler-case
+          (let ((v (coerce (type-error-datum c) (type-error-expected-type c))))
+            (invoke-restart r v))
+        (error ())))))
+→
+TYPE-ERROR-AUTO-COERCE
+(let ((x 3))
+  (handler-bind ((type-error #'type-error-auto-coerce))
+    (check-type x float) x))
+→
+3.0
+See Also
+{\secref\Restarts}, {\secref\InterfacesToRestarts}, Function INVOKE-RESTART, Function STORE-VALUE, Macro CCASE, Macro CHECK-TYPE, Macro CTYPECASE, Function USE-VALUE, Restart USE-VALUE
\ No newline at end of file
